omnibus-sepolia-carina.toml

```
@@ -0,0 +1,539 @@
+
+[setting.wsteth_skew_scale]
+defaultValue = "<%= parseEther(String(1_000_000)) %>"
+
+# Global market configurations
+[setting.min_keeper_fee_usd]
+defaultValue = "<%= parseEther('15') %>"
+
+[setting.max_keeper_fee_usd]
+defaultValue = "<%= parseEther('500') %>"
+
+[setting.keeper_profit_margin_usd]
+defaultValue = "<%= parseEther('20') %>"
+
+[setting.keeper_profit_margin_percent]
+defaultValue = "<%= parseEther('0.2') %>"
+
+[setting.collateral_discount_scalar]
+defaultValue = "<%= parseEther('1') %>"
+
+[setting.min_collateral_discount]
+defaultValue = "<%= parseEther('0.02') %>"
+
+[setting.max_collateral_discount]
+defaultValue = "<%= parseEther('0.2') %>"
+
+[setting.utilization_breakpoint_percent]
+defaultValue = "<%= parseEther('0.80') %>"
+
+[setting.low_utilization_slope_percent]
+defaultValue = "<%= parseEther('0.000025') %>"
+
+[setting.high_utilization_slope_percent]
+defaultValue = "<%= parseEther('0.01') %>"
+
+# ETH market configurations
+[setting.maker_fee]
+defaultValue = "<%= parseEther('0.0001') %>"
+
+[setting.taker_fee]
+defaultValue = "<%= parseEther('0.0006') %>"
+
+[setting.max_market_size]
+defaultValue = "<%= parseEther(String(1000)) %>"
+
+[setting.max_funding_velocity]
+defaultValue = "<%= parseEther('3') %>"
+
+[setting.skew_scale]
+defaultValue = "<%= parseEther(String(1_000_000)) %>"
+
+[setting.funding_velocity_clamp]
+defaultValue = "<%= parseEther('0.0003') %>"
+
+[setting.min_credit_percent]
+defaultValue = "<%= parseEther('0.25') %>"
+
+[setting.min_margin_usd]
+defaultValue = "<%= parseEther('1000') %>"
+
+[setting.min_margin_ratio]
+defaultValue = "<%= parseEther('0.05') %>"
+
+[setting.incremental_margin_scalar]
+defaultValue = "<%= parseEther('1') %>"
+
+[setting.maintenance_margin_scalar]
+defaultValue = "<%= parseEther('0.5') %>"
+
+[setting.max_initial_margin_ratio]
+defaultValue = "<%= parseEther('0.9') %>"
+
+[setting.liquidation_reward_percent]
+defaultValue = "<%= parseEther('0.0001') %>"
+
+[setting.liquidation_limit_scalar]
+defaultValue = "<%= parseEther('1') %>"
+
+[setting.liquidation_max_pd]
+defaultValue = "<%= parseEther('0.0006') %>"
+
+[setting.liquidation_window_duration]
+defaultValue = "36"
+
+# --- Provision bfp market --- #
+#
+#
+
+[provision.bfp_market_factory]
+source = "<%= settings.bfp_market_package %>"
+targetPreset = "<%= settings.target_preset %>"
+options.coreProxyOwner = "<%= settings.owner %>"
+options.owner = "<%= settings.owner %>"
+options.salt = "<%= settings.salt %>"
+options.synthetixPackage = "<%= settings.snx_package %>@<%= settings.target_preset %>"
+options.synthetixPreset = "<%= settings.target_preset %>"
+depends = ["provision.system"]
+
+# --- Configure oracles for markets --- #
+#
+#
+
+# wstETH/stETH oracle
+[provision.wsteth_to_steth_ratio_oracle]
+source = "<%= settings.wsteth_to_steth_ratio_oracle %>"
+targetPreset = "<%= settings.target_preset %>"
+options.wsteth = "<%= settings.wsteth_address %>"
```

maybe need owner and salt as well?
```suggestion
options.owner = "<%= settings.owner %>"
options.salt = "<%= settings.salt %>"
options.wsteth = "<%= settings.wsteth_address %>"
```